SU2_CFD/include/fluid/CFluidModel.hpp

```diff
@@ -147,7 +147,10 @@ class CFluidModel {
    * \brief Evaluate data manifold for flamelet or data-driven fluid problems.
    * \param[in] input - input data for manifold regression.
    */
-  virtual unsigned long EvaluateDataSet(const vector<su2double> &input_scalar, unsigned short lookup_type, vector<su2double> &output_refs) { return 0; }
+  virtual unsigned long EvaluateDataSet(const vector<su2double>& input_scalar, unsigned short lookup_type,
+                                        vector<su2double>& output_refs) {
+    return 0;
+  }
 
```

Comment:
`virtual unsigned long EvaluateDataSet(const vector<su2double> &input_scalar, unsigned short lookup_type, vector<su2double> &output_refs) { return 0; }`

I am not sure why the formatting has changed.

Reply:
What do you mean exactly?

Reply:
I was suggesting an inline implementation of the functions, similar to what it was in the past.

`virtual unsigned long EvaluateDataSet(const vector<su2double> &input_scalar, unsigned short lookup_type, vector<su2double> &output_refs) { return 0; }`

instead of how it is right now which is over multiple lines.